frontend/tests/unit/models/transportation-analysis-test.js

```diff
@@ -1,13 +1,66 @@
+    assert.equal(analysis.offStreetParkingSpaces, 170) // project.parkingLandUse[0].spaces + project.parkingLandUse[1].spaces (name: Garages, name: Lots)
+
+    assert.equal(analysis.residentialUnitsRatio, 5); // residentialUnitsRatio = residentialUnits / thresholdFor residentialUnits zone 2 = 1000/200
+    assert.equal(analysis.officeSqFtRatio, 0.00015); // officeSqFtRatio = officeSqFt / thresholdFor officeSqFtRatio zone 2 = 15/100000
+    assert.equal(analysis.regionalRetailSqFtRatio, 0.00175); // regionalRetailSqFtRatio = regionalRetailSqFt / thresholdFor regionalRetailSqFt zone 2 = 35/20000
+    assert.equal(analysis.localRetailSqFtRatio, 0.004); // localRetailSqFtRatio = localRetailSqFt / thresholdFor localRetailSqFt zone 2 =  60/15000
+    assert.equal(analysis.restaurantSqFtRatio, 0.0045); // restaurantSqFt Ratio = restaurantSqFt / thresholdFor restaurantSqFt zone 2 = 90/20000
+    assert.equal(analysis.communityFacilitySqFtRatio, 0.004); // communityFacilitySqFtRatio = communityFacilitySqFt / thresholdFor communityFacilitySqFt zone 2 =  100/25000
+    assert.equal(analysis.offStreetParkingSpacesRatio, 2); // offStreetParkingSpacesRatio = offStreetParkingSpaces / thresholdFor offStreetParkingSpaces zone 2 =  170/85
+  });
+
+  test('it correctly calculates weightedAverage', async function(assert) {
+    let analysisMirage = server.create('transportation-analysis', 'projectLandUse');
+    let project = await this.owner.lookup('service:store').findRecord(
+      'project', analysisMirage.projectId, { include: 'transportation-analysis' }
+    );
+    let analysis = await project.get('transportationAnalysis');
+
+    const sum = 5 + 0.00015 + 0.00175 + 0.004 + 0.0045 + 0.004 + 2; // sum = ~7.0144
+
+    assert.equal(analysis.weightedAverage, sum)
+    assert.equal(analysis.weightedAverageOver1, true) // if weightedAverage > 1, then true
+  });
+
+  test('it correctly calculates hasFastFood and hasCommunityFacility', async function(assert) {
+    let analysisMirage = server.create('transportation-analysis', 'projectLandUse');
+    let project = await this.owner.lookup('service:store').findRecord(
+      'project', analysisMirage.projectId, { include: 'transportation-analysis' }
+    );
+    let analysis = await project.get('transportationAnalysis');
+
+    assert.equal(analysis.hasFastFood, true);
+    assert.equal(analysis.hasCommunityFacility, true);
+  });
+
+  test('it correctly calculates detailedAnalysis', async function(assert) {
+    let analysisMirage = server.create('transportation-analysis', 'projectLandUse');
+    let project = await this.owner.lookup('service:store').findRecord(
+      'project', analysisMirage.projectId, { include: 'transportation-analysis' }
+    );
+    let analysis = await project.get('transportationAnalysis');
+
+    assert.equal(analysis.detailedAnalysis, true); // true if hasFastFood OR hasCommunityFacility OR weightedAverageOver1 is true (all 3 are true in this case)
```

Comment:
should assert that a model with each of those individual conditions correctly sets `detailedAnalysis`